### PR TITLE
Modal and Spinner Positioning

### LINKED ISSUE #112: fix positioning for pokeball loader spinner
**Is your feature request related to a problem? Please describe.**
reposition pokeball loader spinner inside modal and anywhere else needed.

### LINKED ISSUE #115: center modal for smartphone responsive designs
**Is your feature request related to a problem? Please describe.**
Modal is off centered in responsive mobile for 375px and 414px widths. 

image

### PR DESCRIPTION
**Pull request by Gjhnsn**
## Changes
1. Centered modal for mobile devices
2. Centered spinner in modal for desktop and mobile

## Purpose
Modal was off center for mobile screens.
The spinner in the modal was not centered for desktop or mobile.

Closes #112 
Closes #115

### CODE REVIEW

File: src/components/BaseStats.js [hunk 1]
```diff
@@ -12,79 +12,80 @@ const BaseStats = ({ stats, types }) => {
...
-                backgroundColor: [getTypeColors[typeColor]],
-                hoverBackgroundColor: [getTypeColors[typeColor]],
-                borderRadius: 50,
-                borderSkipped: false,
-                borderWidth: 1,
-                barThickness: 16,
+        {/* show bar graph or spinner by checking for baseStatsArray */}
+        {baseStatsArray ? (
+          <Bar
+            placeholder={"hi"}
```
Comment (Lap343) [inline src/components/BaseStats.js:18]: hi

File: src/components/BaseStats.js [hunk 2]
```diff
@@ -12,79 +12,80 @@ const BaseStats = ({ stats, types }) => {
...
                   display: false,
-                  drawBorder: false,
+                },
+                title: {
+                  display: false,
+                  text: "",
+                },
+                tooltip: {
+                  enabled: true,
+                  backgroundColor: "rgb(0,0,0,0.6",
```
Comment (BlackCubes) [inline src/components/BaseStats.js:78]: Might not be a big deal, but it looks like you're missing an ending parenthesis?

File: src/components/PokeList.js [hunk 3]
```diff
@@ -181,42 +180,48 @@ const PokeList = ({
...
+            <InfiniteScroll
+              className="card-container"
+              loader={<Spinner />}
+              dataLength={allPokemons.length}
+              hasMore={true}
+              scrollThreshold={0.8}
+              next={getMorePokemons}
+              endMessage={
+                <p style={{ textAlign: "center" }}>
+                  <b>Yay! You have seen it all</b>
```
Comment (BlackCubes) [inline src/components/PokeList.js:206]: :O

File: src/components/BaseStats.js (hunk 2 above)
Comment (kmausolf) [inline src/components/BaseStats.js:78]: I think you might be missing a closing paren here.

File: src/components/EvolutionTab.js [hunk 4]
```diff
@@ -136,15 +134,15 @@ const EvolutionTab = ({ evolutionChainUrl, pokemonColor }) => {
...
-          overflowX: "auto"
+          overflowX: "auto",
           // ** unknown solution for safari and chrome **
         }}
-        className={"spriteContainer"}
-
-      > {spritesWithEvolutionArrows()}</div>
-
+        className={"spriteContainer"}>
+        {" "}
```
Comment (kmausolf) [inline src/components/EvolutionTab.js:141]: What might this be for?

**Review by Lap343 — APPROVED** [review]
After those fixes already commented on I think this code is ready to go

**Review by JorgeLVilla — APPROVED** [review]
Aside from the comments of the couple missing things, everything looks good. Nice job Garret!

File: src/components/EvolutionTab.js (hunk 4 above)
Comment (krferrell) [reply src/components/EvolutionTab.js:141]: Prettier does this! Not sure why.

File: src/components/BaseStats.js (hunk 2 above)
Comment (Gjhnsn) [reply src/components/BaseStats.js:78]: thanks! fixing!

**Pull request merged**